osquery/core/tables.cpp

```diff
@@ -430,7 +430,7 @@ std::set<std::string> ConstraintList::getAll(ConstraintOperator op) const {
 }
 
 template <typename T>
-std::set<T> ConstraintList::getAll(ConstraintOperator op) const {
+std::set<T> ConstraintList::getAll(ConstraintOperator op /* unused */) const {
```

Comment:
Shouldn't this be `(ConstraintOperator /* op */)` instead?

Reply:
Nice catch. Thanks 👍